armsrc/Standalone/hf_bog.c

```diff
@@ -0,0 +1,278 @@
+//-----------------------------------------------------------------------------
+// This code is licensed to you under the terms of the GNU GPL, version 2 or,
+// at your option, any later version. See the LICENSE.txt file for the text of
+// the license.
+//-----------------------------------------------------------------------------
+// main code for standalone HF Sniff (and ULC/NTAG/ULEV1 pwd storing)
+//-----------------------------------------------------------------------------
+#include "hf_bog.h"
+
+#define DELAY_READER_AIR2ARM_AS_SNIFFER (2 + 3 + 8) 
+#define DELAY_TAG_AIR2ARM_AS_SNIFFER (3 + 14 + 8)
+
+// Maximum number of auth attempts per standalone session
+#define MAX_PWDS_PER_SESSION 20
+
```

Comment:
only 20?   20*4 == 80 ..  Try 256/4= 64?

Reply:
Yep, I increased it.